plugins/modules/simpleinit_msb.py

```diff
@@ -0,0 +1,300 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2016-2023, Vlad Glagolev <scm@vaygr.net>
+#
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: simpleinit_msb
+short_description: Manage services on Source Mage GNU/Linux
+version_added: 7.1.0
+description:
+    - Controls services on remote hosts using C(simpleinit-msb).
+author: "Vlad Glagolev (@vaygr)"
+extends_documentation_fragment:
+    - community.general.attributes
+attributes:
+    check_mode:
+        support: full
+    diff_mode:
+        support: none
+options:
+    name:
+        type: str
+        description:
+            - Name of the service.
+        required: true
+        aliases: ['service']
+    state:
+        type: str
+        required: false
+        choices: [ running, started, stopped, restarted, reloaded ]
+        description:
+          - C(started)/C(stopped) are idempotent actions that will not run
+            commands unless necessary.  C(restarted) will always bounce the
+            service.  C(reloaded) will always reload. B(At least one of state
+            and enabled are required.) Note that reloaded will start the
+            service if it is not already started, even if your chosen init
+            system would not normally.
+    enabled:
+        type: bool
+        required: false
+        description:
+        - Whether the service should start on boot. B(At least one of state and
+          enabled are required.)
+'''
+
+EXAMPLES = '''
+# Example action to start service httpd, if not running
+- community.general.simpleinit_msb:
+    name: httpd
+    state: started
+
+# Example action to stop service httpd, if running
+- community.general.simpleinit_msb:
+    name: httpd
+    state: stopped
+
+# Example action to restart service httpd, in all cases
+- community.general.simpleinit_msb:
+    name: httpd
+    state: restarted
+
+# Example action to reload service httpd, in all cases
+- community.general.simpleinit_msb:
+    name: httpd
+    state: reloaded
+
+# Example action to enable service httpd, and not touch the running state
+- community.general.simpleinit_msb:
+    name: httpd
+    enabled: yes
```

Comment:
```suggestion
    enabled: true
```
Always use `true` and `false` for YAML booleans.